runtime/BUILD

```diff
@@ -72,23 +72,23 @@ cc_test(
     ],
 )
 
-cc_binary(
+cc_test(
     name = "buffer_benchmark",
     srcs = ["buffer_benchmark.cc"],
     deps = [
         ":runtime",
         "@google_benchmark//:benchmark_main",
     ],
-    testonly = 1,
+    args=["--benchmark_min_time=0.1s"],
```

Comment:
...I didn't know `args` was a thing, will try using it to solve some other problems...